### DISCUSSION TITLE
2.0组织结构图的问题

### DISCUSSION THREAD

**Discussion by plutons** [ant-design/ant-design-charts#2737] [Q&A] [answered]
https://ant-design-charts.antgroup.com/zh/examples/relations/organization-chart/#default
这个例子是从@ant-design/graphs导入的OrganizationChart
`import { OrganizationChart } from '@ant-design/graphs';`

但是我导入的时候ts报错OrganizationChart 不存在，请问要如何解决呢？

` "@ant-design/charts": "^2.2.1",
    "@ant-design/graphs": "^1.4.2",`
依赖是这样的

**Comment by yvonneyx** [comment, marked as answer]
本周已发 Graphs 2.0 正式版，可以使用了哦

**Reply by ideaviewes** [reply]
@yvonneyx 感谢你们的辛勤付出

**Reply by susanwei126** [reply]
> <img alt="" width="32" height="32" src="https://avatars.githubusercontent.com/u/55794321?s=64&amp;v=4">
> yvonneyx

请问同学能不能看下issues 2.0里面很多很明显的bug 可不可以修复一下 这么多明显的bug都不敢用2.0了。。。

**Reply by yvonneyx** [reply]
> > <img alt="" width="32" height="32" src="https://avatars.githubusercontent.com/u/55794321?s=64&amp;v=4">
> > yvonneyx
> 
> 请问同学能不能看下issues 2.0里面很多很明显的bug 可不可以修复一下 这么多明显的bug都不敢用2.0了。。。

Plots 还是 Graphs 呢？可以在具体 issue 下艾特我们，会找对应的 maintainer 跟进

**Reply by susanwei126** [reply]
> Graphs

plot！！！chart！！！ 感恩！请尽快修复！
https://github.com/ant-design/ant-design-charts/issues/2831
https://github.com/ant-design/ant-design-charts/issues/2830
https://github.com/ant-design/ant-design-charts/issues/2832

主要问题集中在 ，单击/双击element事件，以及分组之后tooltip的展示问题
